q2_sample_classifier/plugin_setup.py

```diff
@@ -249,11 +248,11 @@
         },
     output_descriptions={
         'predictions': 'leave one out predictions for each sample',
-        'model_summary': 'number of neighbors k',
         'accuracy_results': 'Accuracy results visualization.',
     },
-    name=('Train and test a cross-validated (leave one out)'
-          ' supervised learning classifier.'),
+    name=('Run k-nearest-neighbors on a labeled distance matrix.'
```

Comment:
oh the "name" looked fine, though this looks better!

My comment about the description referred to the "description" below — you can more or less just replicate the "name" in the "description". Currently, the description line below prints this in the help docs:
> Predicts a categorical sample metadata column using a supervised learning
  classifier. Splits input data into training and test sets. The training
  set is used to train and test the estimator using a stratified k-fold
  cross-validation scheme. This includes optional steps for automated
  feature extraction and hyperparameter optimization. The test set validates
  classification accuracy of the optimized estimator. Outputs classification
  results for test set. For more details on the learning algorithm, see
  http://scikit-learn.org/stable/supervised_learning.html